app/web/template/sign_your_lpa.gohtml

```diff
@@ -11,31 +11,22 @@
 
       {{ trFormatHtml .App "signYourLpaHeader" }}
 
-            <p class="govuk-body govuk-!-font-weight-bold">
-                {{ trFormatCount .App "wantAttorneysToBeAppointed" (len .Lpa.Attorneys) "AllAttorneysFullNames" .Lpa.AttorneysFullNames "LpaType" (tr .App .Lpa.TypeLegalTermTransKey) }}
-            </p>
-
-      {{ if eq .Lpa.Type "pfa" }}
-        {{ trFormatHtml .App "explainAttorneysPowersPfa" "AllAttorneysFirstNames" .Lpa.AttorneysFirstNames }}
-      {{ else }}
-        {{ trFormatHtml .App "explainAttorneysPowersHw" "AllAttorneysFirstNames" .Lpa.AttorneysFirstNames }}
-      {{ end }}
-
-      <div class="govuk-warning-text">
-        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-        <strong class="govuk-warning-text__text">
-          <span class="govuk-warning-text__assistive">{{ tr .App "warning" }}</span>
-          {{ trFormat .App "attorneysCanActOnlyIfAskedWarning" "AllAttorneysFirstNames" .Lpa.AttorneysFirstNames }}
-        </strong>
-      </div>
-
-      {{ trFormatHtml .App "attorneysMustFollowMCA" }}
-
       {{ template "details" (details . "whatHappensIfIWantToMakeChanges" "whatHappensIfIWantToMakeChangesContent" false) }}
 
-      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+      <p class="govuk-body"></p>
+
+      {{ $lpaTypeLegalTerm := tr .App .Lpa.TypeLegalTermTransKey }}
 
-      {{ trHtml .App "formPanelContent" }}
+      <ul class="govuk-list govuk-list--bullet">
+        <li>{{ trFormat .App "bySigningIConfirm" "TypeLegalTermTransKey" $lpaTypeLegalTerm }}</li>
```

Comment:
<img width="701" alt="image" src="https://user-images.githubusercontent.com/146390/216975331-76e73912-ba5c-4583-8c98-a1ed8dd2fc84.png">

This isn't in the list on the design


![image](https://user-images.githubusercontent.com/146390/216975378-073125e2-1a92-44c5-903a-0456d558c4a0.png)

Reply:
My bad - I'd got the `p` ready but skipped over putting the translation in.